var/spack/repos/builtin/packages/r-dendextend/package.py

```diff
@@ -9,15 +9,17 @@
 class RDendextend(RPackage):
     """dendextend: Extending 'Dendrogram' Functionality in R"""
 
-    homepage = "https://CRAN.R-project.org/package=dendextend"
-    url      = "https://cran.r-project.org/src/contrib/dendextend_1.5.2.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/dendextend"
+    homepage = "https://cloud.r-project.org/package=dendextend"
+    url      = "https://cloud.r-project.org/src/contrib/dendextend_1.5.2.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/dendextend"
 
+    version('1.12.0', sha256='b487fed8c1878a23b9e28394ee11f16a1831b76c90793eb486e6963c7162fa55')
+    version('1.10.0', sha256='88f0fb3362d69144daf4f35d0ea09f32c2df1adf614e040327a42552a8fd3224')
     version('1.5.2', '1134869d94005727c63cf3037e2f1bbf')
 
-    depends_on('r@3.0.0:')
+    depends_on('r@3.0.0:', type=('build', 'run'))
     depends_on('r-magrittr@1.0.1:', type=('build', 'run'))
     depends_on('r-ggplot2', type=('build', 'run'))
     depends_on('r-fpc', type=('build', 'run'))
```

Comment:
This is either no longer a dependency or never was.

Reply:
It was a dependency up until version 1.10.0. I added that constraint.